README.md

```diff
@@ -30,15 +30,21 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
-1. What is the DOM?
+1. What is the DOM? 
+
+the DOM is the document object model, which allows javascipt to access and modify an html document.
```

Comment:
It's not to modify the html document directly, but it is just a representation of the html document on the browser